conconartist/travel-tracker
# Feature/connect display

#### What's this PR do? 

Built display functionality to the DOM fetch requests of corresponding APIs.  User should see their trips categorized by present, pending, past, and upcoming.  The total amount spent for the year 2020 is displayed.  The existing Traveler class has also been added to with methods to categorize the user's trips. 

#### Where should the reviewer start? 

Go to Traveler.js to see the changes.  Changes have been made to domUpdates.js and index.js to reflect the updates.

#### How should this be manually tested?

Clone the repo and connect to the local server with the webpack API before entering `npm start` in your terminal for both the local server and the project repo.  Go to http://localhost: 8080 to see the changes from the user view.

#### Issues:

There was some difficulty figuring out an appropriate way to calculate "present" trips to take into consideration the duration of the trip, not just the date.  In the future, moment.js might help alleviate the possible hiccups from finding the dates of the trip's duration. 

 #### Screenshots (if appropriate)

<img width="927" alt="Screen Shot 2021-01-18 at 4 32 24 PM" src="https://user-images.githubusercontent.com/67291333/104971156-25007a00-59ab-11eb-94be-7dace2d52030.png">
